proto/lbm/foundation/v1/tx.proto

```diff
@@ -50,6 +50,9 @@ service Msg {
   // Revoke revokes any authorization corresponding to the provided method name on the
   // granter that has been granted to the grantee.
   rpc Revoke(MsgRevoke) returns (MsgRevokeResponse);
+
+  // OneTimeMint defines a one-time mint coins to the treasury.
+  rpc OneTimeMint(MsgOneTimeMint) returns (MsgOneTimeMintResponse);
```

Comment:
Please add cli and cli unittest.

Reply:
In the case of one-time mint, it is not appropriate to expose it to the cli because it should be performed only through proposals.

Reply:
The other proposal tx also has cli and cli test. For example, `WithdrawFromTreasury` and `Grant`. Check it please.

Reply:
I added cli and cli test. Thank you.